lib/trino-orc/src/test/java/io/trino/orc/TestOrcBloomFilters.java

```diff
@@ -276,6 +278,49 @@ public void testMatches()
         assertTrue(emptyPredicate.matches(1L, matchingStatisticsByColumnIndex));
     }
 
+    @Test
+    public void testMatchesExpandedRange()
```

Comment:
Let's have a test which verifies that the threshold value is also honoured

Reply:
Oops, you're right - I forgot it...
Adding now :)